virtcontainers/pkg/cgroups/manager.go

```diff
@@ -331,3 +331,16 @@ func (m *Manager) RemoveDevice(device string) error {
 	m.Unlock()
 	return fmt.Errorf("device %v not found in the cgroup", device)
 }
+
+func (m *Manager) UpdateCpuSets(cpuset string) error {
```

Comment:
`SetCpuSets` makes more sense to me

Reply:
Sure.  Added reference to https://github.com/kata-containers/runtime/issues/2811 in the code.